R/aux_functions.R

```diff
@@ -55,9 +58,11 @@ extinct_prob <- function(outbreak_df_week = NULL, cap_cases  = NULL, week_range
 
 #' Calculate whether outbreaks went extinct or not
 #' @author Joel Hellewell
-#' @param outbreak_df_week `data.table` weekly cases produced by the outbreak model
+#' @param outbreak_df_week a `data.table`: weekly cases produced by the
+#'   outbreak model
 #' @inheritParams outbreak_model
-#' @param week_range integer vector giving the (zero indexed) week range to test for whether an extinction occurred.
+#' @param week_range a positive `integer` vector: giving the (zero indexed)
```

Comment:
is this a full-vector-vector, or a length-two-vector? docs read like length 2?

Reply:
Thanks for pointing this out. Will address in a subsequent PR.